data/config/ANDAO.js

```diff
@@ -1,30 +1,30 @@
 const rpc = {
   agent:            undefined,                                          // Set with the agent app of the AN DAO deployed to Aragen
-  votingApp:        undefined,                                          // Set with the voting app of the AN DAO deployed to Aragen
+  voting:           undefined,                                          // Set with the voting app of the AN DAO deployed to Aragen
   tokenManager:     undefined,                                          // Set with the token manager of the AN DAO deployed to Aragen
 }
 
 const ropsten = {
   agent:            undefined,
-  votingApp:        undefined,
+  voting:           undefined,
   tokenManager:     undefined,
 }
 
-const staging = {
+const staging = {   // https://rinkeby.aragon.org/#/andaostaging
   agent:            '0xec5700f0774cc9e7f2d89bee4b14401124fc2d06',       // Agent of AN DAO Staging
-  votingApp:        '0x5505f0db0ff6990459e812e19bf7895148c6a8b3',       // Voting of AN DAO Staging
+  voting:           '0x5505f0db0ff6990459e812e19bf7895148c6a8b3',       // Voting of AN DAO Staging
   tokenManager:     '0x2cedb6250da2e4798f742eb60b075c8f4ef5ef6a',       // Token Manager of AN DAO Staging
 }
 
-const rinkeby = {
+const rinkeby = {   // https://rinkeby.aragon.org/#/0x2eBe0f5854Fa5C685DAd156Dd8A07562eD1F74b1
```

Comment:
We can assign a name to this DAO if you'd like

Reply:
we did, don't know why we linked it that way, in some of the PRs I already changed it to `https://rinkeby.aragon.org/#/andao`